segregation/util/util.py

```diff
@@ -2,14 +2,15 @@
 Useful functions for segregation metrics
 """
 
-__author__ = "Levi Wolf <levi.john.wolf@gmail.com> and Renan X. Cortes <renanc@ucr.edu>"
+__author__ = "Levi Wolf <levi.john.wolf@gmail.com>, Renan X. Cortes <renanc@ucr.edu>, and Eli Knaap <ek@knaaptime.com>"
 
 import numpy as np
 import pandas as pd
 import libpysal
 import geopandas as gpd
+import math
 from warnings import warn
-from libpysal.weights import Queen, KNN, Kernel
+from libpysal.weights import Kernel
```

Comment:
I think `_return_length_weighted_w` uses KNN, could we import here?

Reply:
we're using the full path from libpysal
```w_aux = libpysal.weights.KNN.from_dataframe(
            data,
            ids=data.index.tolist(),
            geom_col=data._geometry_column_name,
            k=1)
```
so we dont need to import KNN again

Reply:
It makes sense!